Require consent rather than 2/3 vote for emeritus (#717)
openjs-foundation/cross-project-council: PROJECT_PROGRESSION.md

#### Acceptance Criteria

> Projects may be granted Emeritus status with the consent of the CPC and with approval from project ownership.

Reviewer: Should this be "consensus" rather than consesnt?

Reviewer 2: Suggesting the following wording to make your life easier. Feel free to ignore and word differently.

```suggestion
Projects may be granted Emeritus status through CPC consensus and with the approval of the maintainers of the project.
```

> In cases where there is a lack of project ownership, a 2/3 vote from the CPC is required.

Reviewer 2: Clarifying the concept of "ownership" which isn't really defined anywhere:

```suggestion
In cases where the project maintainers aren't responding to the CPC for six (6) weeks after having been contacted through the appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
```

Author: I think I agree with this in general, but it feels like a slightly bigger change to add this level of specificity. Also, is it intentional that you're proposing dropping the 2/3 vote requirement for the no-maintainers/owners case?

Reviewer 2: > I think I agree with this in general, but it feels like a slightly bigger change to add this level of specificity.

Would you prefer:

```suggestion
In cases where the project maintainers aren't responding after having been repeatedly contacted through appropriate channels about the CPC's intent to move the project to Emeritus stage, the CPC may proceed with the stage change without approval from the maintainers.
```

> Also, is it intentional that you're proposing dropping the 2/3 vote requirement for the no-maintainers/owners case?

Yes. I don't see the rationale for making a distinction between those two cases. If anything, Emeritus should be automatic for projects who are no longer maintained.
